README.md

```diff
@@ -11,60 +11,70 @@ on IBM Blockchain.  The components of the demo are:
 * A Trade Center for buying and selling existing trades.
 * A special interface just for auditors of the network to examine trades
 
+##### Versions and Supported Platforms
+On November 9th, 2016, we released the IBM Blockchain Service v1.0 based on HyperLedger fabric v0.6.  All new networks created in bluemix will be this version.  Support of the v0.4.2.x Bluemix Service based on the 0.5.3 Hyperledger Fabric has been deprecated.  It is strongly recommended that if you have an existing network based on 0.5.3, you redeploy a new network and follow the instructions in the 2.0 branch.
+
+- [CP-Web - Branch v2.0](https://github.com/ibm-blockchain/cp-web/tree/v2.0)
+	- Works with Hyperledger fabric `v0.6-developer-preview`
```

Comment:
Yes, you did mention it in the text.  I was wondering if we should also put `IBM Bluemix Blockchain Service v1.0.0+` or something like that in the list, like the list below it.  I guess it's a minor detail and shouldn't hold up the PR.